pkg/utils/validation/cidr/cidr_test.go

```diff
@@ -184,6 +189,30 @@ var _ = Describe("cidr", func() {
 			})
 		})
 
+		Context("ValidateCIDRIPFamily", func() {
```

Comment:
The func that is being tested in this context is `ValidateIPFamily`.

```suggestion
		Context("ValidateIPFamily", func() {
```